app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -21,20 +21,20 @@
     </div>
     <div class="field">
       <label class="required">{{t 'Tax name'}}</label>
-      <Input @type="text" @id="tax_name" @value={{this.tax.name}} />
+      <Input @type="text" @autocomplete="some-random-string" @id="tax_name" @value={{this.tax.name}} />
```

Comment:
Remove this

Reply:
@iamareebjamal should I remove the whole autocomplete attribute from here only?

Reply:
The issue is about country field, so why add this here?

Reply:
@iamareebjamal Sir I have to add this autocomplete to every input field, to prevent it from getting auto-filled, and if I am removing it from there then auto-filling is not disabled for those fields
![confirm](https://user-images.githubusercontent.com/62847072/95680104-aef2b700-0bf4-11eb-8579-98c5255cd78a.png)
 i.e. all information will get auto-filled like it is behaving now in eventyay.com

Reply:
As I understand, the issue is about country autofill disabling, not all fields

Reply:
Actually sir I think should be the other way round as stated by @mariobehling, because we want that a person adding his or her tax info should not have the option to autofill all the fields based on the browsers form's memory that's why we want to disable it and only allow the autofilling of country name, and not other sensitive information that is user-specific.

Reply:
@mariobehling There is still confusion. Do we want to?

- [ ] 1. Disable all browser autocomplete including name, address, etc
- [ ] 2. Disable browser autocomplete for country and allow browser autocomplete for other form fields
- [ ] 3. Disable browser autocomplete for all fields and allow browser autocomplete for country field

Our suggestion autocomplete will be shown in all 3 cases

Reply:
Thanks for clarifying. We need:
* Disable browser autocomplete for country and allow browser autocomplete for other form fields

Reply:
@ShreyanshSrivastava Please remove the autocomplete tag from other fields

Reply:
@iamareebjamal Done